tests/RecordTests/UnitTest/playback_ut.cpp

```diff
@@ -93,7 +93,7 @@ TEST_F(playback_ut, open_large_file)
     k4a_stream_result_t stream_result = K4A_STREAM_RESULT_FAILED;
```

Comment:
I don't see any tests in this file that just validate inputs for the API's. Can you add that?

Reply:
I'm not entirely sure what you mean by "validate inputs". Are you talking about testing all the `RETURN_VALUE_IF_ARG()` cases? Or something else.

Reply:
Tests that validate the inputs to your new functions. Like making sure passing in a NULL handle is a failure and not a crash. There should be tests that validate the full range of inputs each function can see.

Reply:
I think I've covered all the cases for valid inputs, as well as a couple of the less-obvious invalid input cases.
Testing things like NULL handle / input arguments for each function seems like a lot of extra code for not a lot of gain, considering every function already starts with something like `RETURN_VALUE_IF_HANDLE_INVALID(K4A_RESULT_FAILED, k4a_playback_t, playback_handle);`.
These cases are very unlikely to break in the future from code changes, and would have fairly low impact even if they don't immediately return.

I can go through and add tests for the full input space of each function, but that will have to wait until later next week, since I've got other things to work on in the mean time.